src/runtime/server.ts

```diff
@@ -73,20 +70,19 @@ async function sandboxStarted(port: number, timeout: number = 20_000): Promise<v
   throw new Error(`Sandbox Server with port: ${port} failed to start after ${timeout}ms`);
 }
 
+function initalPort(): number {
+  return Math.floor(Math.random() * 10000);
```

Comment:
I think we should probably set the floor to [1024 or thereabouts](https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers)

Reply:
True, this was when I wanted to make sure that they didn't overlap folders.